src/pybind/mgr/dashboard/frontend/src/app/shared/components/grafana/grafana.component.ts

```diff
@@ -73,26 +107,16 @@ export class GrafanaComponent implements OnInit, OnChanges {
   }
 
   getFrame() {
-    this.url = this.baseUrl + this.grafanaPath + '&refresh=2s' + this.mode;
+    this.url = this.baseUrl + this.grafanaPath + '&refresh=2s' + this.mode + '&' + this.time;
     this.grafanaSrc = this.sanitizer.bypassSecurityTrustResourceUrl(this.url);
   }
 
-  timePickerToggle() {
-    this.modeFlag = true;
-    this.mode = this.mode ? '' : '&kiosk';
-    if (this.modeText === 'Return to default') {
-      this.modeText = 'Change time selection';
-      this.reset();
-    } else {
-      this.modeText = 'Return to default';
-    }
+  ontpChange(event) {
     this.getFrame();
-    this.modeFlag = false;
   }
 
   reset() {
-    this.mode = '&kiosk';
-    this.modeText = 'Change time selection';
+    this.time = 'from=now-1h&to=now';
```

Comment:
Given this string is repeated twice, why not moving it to a constant `readonly DEFAULT_TIME: string = 'from=now-1h&to=now'`?

Reply:
ack